graspologic/embed/svd.py

```diff
@@ -205,6 +206,10 @@ def selectSVD(X, n_components=None, n_elbows=2, algorithm="randomized", n_iter=5
             Does not support ``graph`` input of type scipy.sparse.csr_matrix
         - 'truncated'
             Computes truncated svd using :func:`scipy.sparse.linalg.svds`
+        - 'square'
```

Comment:
thoughts on just calling this `eigsh` (question for anyone)? feels more clear to me. or other ideas?  could have called it `square-hermitian-eigendecomposition` (catchy right?) so square feels a bit arbitrary to me

Reply:
yeah, fine with me, c612762